### ISSUE TITLE
Topic operations without privileges

### ISSUE THREAD

**Issue by dariocazas** [adevinta/zoe#20]
Some basic operations available in zoe 0.24.0 gets an auth error in versions 0.25.0 and 0.26.0, like:
- List topics
- Describe topics

Using the following scripts:

```sh
# Test zoe 0.24.0
sudo rpm -e zoe
ZOE_VERSION=0.24.0
sudo rpm -i "https://github.com/adevinta/zoe/releases/download/v${ZOE_VERSION}/zoe-${ZOE_VERSION}-1.x86_64.rpm"
zoe topics list

# Test zoe 0.25.0
sudo rpm -e zoe
ZOE_VERSION=0.25.0
sudo rpm -i "https://github.com/adevinta/zoe/releases/download/v${ZOE_VERSION}/zoe-${ZOE_VERSION}-1.x86_64.rpm"
zoe topics list

# Test zoe 0.26.0
sudo rpm -e zoe
ZOE_VERSION=0.26.0
sudo rpm -i "https://github.com/adevinta/zoe/releases/download/v${ZOE_VERSION}/zoe-${ZOE_VERSION}-1.x86_64.rpm"
zoe topics list
```

With those results:
- CASE: use an user (configured in `sasl.jaas.config`) with only privileges about READ and DESCRIBE on several topics:
    - 0.24.0: gets the topics that the configured user has privileges
    - 0.25.0: gets error
    - 0.26.0: gets error
- CASE: use kafka admin user:
    - 0.24.0: gets all non internal topics
    - 0.25.0: gets all non internal topics
    - 0.26.0: gets all non internal topics

The error showed is:
```yml
failure: runner 'local' failed
cause:
  failure: org.apache.kafka.common.errors.TopicAuthorizationException: Topic authorization failed.
  cause:
      failure: Topic authorization failed.
```

About config against kafka server:
```yml
      client.id: "zoe-client"
      sasl.mechanism: "SCRAM-SHA-512"
      sasl.jaas.config: "org.apache.kafka.common.security.scram.ScramLoginModule required username=\"user1\" password=\"user1pass\";"
      security.protocol: "SASL_SSL"
      ssl.truststore.location: "/home/myuser/.zoe/config/kafka_keystore.jks"
      ssl.truststore.password: "changeit"
      ssl.endpoint.identification.algorithm: "https"
```

**Comment by wlezzar** [comment]
Hi @dario

**Comment by wlezzar** [comment]
@dariocazas It it possible for you to give the user you are using the right to describe the configs and see if it works just to confirm the issue?

**Comment by wlezzar** [comment]
@dariocazas the issue is fixed in 0.26.1. You may download this release and try again :+1:

**Comment by dariocazas** [comment]
@wlezzar thx for you quick reply and patch. 

Well, over 0.26.1, when use an user with privileges over certain topics (not for all topics) to run `zoe topics lists`, log show:
```sh
2020-10-10 23:18:17 WARN zoe: unexpected error on config describe request
java.util.concurrent.ExecutionException: org.apache.kafka.common.errors.TopicAuthorizationException: Topic authorization failed.
```
But gets OK result (same topic list as 0.24.0 version).

On another side, if I try to describe a topic, gets TopicAuthorizationException and show info without config:
```sh
> zoe -e dev topics describe test
[...]
2020-10-10 23:14:05 WARN zoe: unexpected error on config describe request
java.util.concurrent.ExecutionException: org.apache.kafka.common.errors.TopicAuthorizationException: Topic authorization failed.
[...]
{"topic":"testTopic","internal":false,"partitions":[0,1],"config":{}}
```

The user used has DESCRIBE allowed.
```sh
> kafka-acls.sh --bootstrap-server $BROKERS --command-config $COMMAND_CONFIG --list --principal User:test_user
[...]
ACLs for principal `User:ofda_training_dev`
Current ACLs for resource `ResourcePattern(resourceType=TOPIC, name=testTopic, patternType=LITERAL)`: 
 	(principal=User:test_user, host=*, operation=READ, permissionType=ALLOW)
	(principal=User:test_user, host=*, operation=DESCRIBE, permissionType=ALLOW)
	(principal=User:test_user, host=*, operation=WRITE, permissionType=ALLOW) 
[...]
```

Review about describe operation using kafka-topic.sh gets auth error too:
```sh
> kafka-topics.sh --bootstrap-server $BROKERS --command-config $COMMAND_CONFIG --describe --topic testTopic
[...]
[2020-10-10 23:43:40,231] ERROR org.apache.kafka.common.errors.ClusterAuthorizationException: Request Request(processor=0, connectionId=XXX, session=Session(User:test_user,XXX), listenerName=ListenerName(SASL_SSL), securityProtocol=SASL_SSL, buffer=null) is not authorized.
```

As conclusion: 
- List topic operation works fine now
- The tools need some additional (and unknown for me) ACL to check the topic configs. 

Good job. Thank you
